### DISCUSSION TITLE
We need an orginization for the bark coding org site.

### DISCUSSION THREAD

**Discussion by rubiidev18alt** [cytrinc/bark-coding#51] [New Features (not block requests)]
title. (also i need the repo name for the org site for this)

**Comment by raynepaws** [comment]
agreed

**Comment by mariocraft987** [comment]
same

**Reply by rubiidev18alt** [reply]
well, what's the repo for the org site? (or did you set up an alt?)

**Reply by mariocraft987** [reply]
https://github.com/bark-create :trollface:

**Reply by rubiidev18alt** [reply]
I made an alternative. https://github.com/Bark-Organization (because create doesn't have any activity + I meant the one on bark-org.vercel.app not bark create)

**Reply by raynepaws** [reply]
gah there's so many organizations for bark

**Comment by raynepaws** [comment]
https://github.com/Bark-Program
@rubiidev18alt @mariocraft987
we already have one
but mariocraft could we get invites pleaase?

**Reply by rubiidev18alt** [reply]
well I'm trying to make the Definitive one for this version of Bark, Bark organization is supposed to be the ONE for this version of the site

**Reply by raynepaws** [reply]
are you getting angry? it's hard to tell through a computer screen
i just don't see the point of making three organizations, that's all

**Reply by mariocraft987** [reply]
@bark-coding, @bark-program, @bark-programming and many more

**Reply by raynepaws** [reply]
let's just use one

**Reply by mariocraft987** [reply]
1
